libs/core/tests/unit_tests/indexing/test_indexing.py

```diff
@@ -630,26 +630,18 @@ def test_incremental_delete(
     }
 
 
-def test_incremental_indexing_with_batch_size(
+def test_incremental_delete_with_same_source(
```

Comment:
Unfortunately, the diff in `test_indexing.py` is very confusing, but `test_incremental_delete_with_same_source` is the only test case I newly created.